sidekick_core/lib/src/dart_runtime.dart

```diff
@@ -14,6 +14,10 @@ class SidekickDartRuntime {
 
   /// The location of the Dart SDK
   Directory get dartSdkPath =>
+      sidekickPackage.directory('build/.cache/dart-sdk/');
+
+  /// The old sdk path in sidekick 0.7.0 (cache without .)
+  Directory get _oldDartSdkPath =>
       sidekickPackage.directory('build/cache/dart-sdk/');
```

Comment:
The breaking part is moving from `cache` to `.cache`. I'll revert that and land it later when we have an update mechanism